Extensions/PlatformBehavior/platformruntimebehavior.ts

```diff
@@ -229,6 +229,19 @@ namespace gdjs {
     static NORMALPLAFTORM = 0;
     static JUMPTHRU = 1;
     static LADDER = 2;
+
+    static isOnPlatformTest(object1, object2, behaviorName) {
+      // Check if the objects exist and share the behavior
+      if (object1 === null || !object1.hasBehavior(behaviorName)) {
+        return false;
+      }
+
+      const behavior1 = object1.getBehavior(behaviorName);
+      return (
+        behavior1.isOnFloor() &&
+        behavior1._onFloor.getFloorPlatform().owner.id === object2.id
```

Comment:
Could we expose this as a method `isOnFloorObject`? Better to avoid PlatformRuntimeBehavior accessing internals of PlatformerObjectRuntimeBehavior.